commit 6ca22ee1257e34ee2b89f2fb354d6c382b8f3e29
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Wed Feb 11 10:48:55 2015 +0200

    mk: fix missing link of librte_vhost in shared, non-combined config
    
    When building shared, non-combined library, librte_vhost does not
    get linked in, causing among other things vhost example to fail
    building to do undefined symbols on linkage.
    
    Signed-off-by: Panu Matilainen <pmatilai@redhat.com>

diff --git a/mk/rte.app.mk b/mk/rte.app.mk
index 95dbb0b..55d989e 100644
--- a/mk/rte.app.mk
+++ b/mk/rte.app.mk
@@ -125,6 +125,10 @@ LDLIBS += -lm
 LDLIBS += -lrt
 endif
 
+ifeq ($(CONFIG_RTE_LIBRTE_VHOST), y)
+LDLIBS += -lrte_vhost -lfuse
+endif
+
 endif # ! CONFIG_RTE_BUILD_COMBINE_LIBS
 
 ifeq ($(CONFIG_RTE_LIBRTE_PMD_PCAP),y)
